recipe/gcc.patch

```diff
@@ -0,0 +1,42 @@
+From 108cb298f50f08407e46b6002ad8d85aebc0b77e Mon Sep 17 00:00:00 2001
+From: Jan Janssen <jan-janssen@users.noreply.github.com>
+Date: Thu, 25 Jun 2020 12:15:42 +0200
+Subject: [PATCH] Remove GCC overwrite
```

Comment:
I think it's sufficient to add `-DGCC_SYSTEM_TYPE=""` to the cmake options to disable this.